arm-network/2017-03-01/swagger/virtualNetwork.json

```diff
@@ -983,6 +1025,70 @@
         }
       },
       "description": "Response for CheckIPAddressAvailability API service call"
+    },
```

Comment:
use the usage object here: https://github.com/Azure/azure-rest-api-specs/blob/master/arm-network/2017-03-01/swagger/usage.json